conf/messages

```diff
@@ -105,13 +105,13 @@ verifyEmail.lineTwo.3 = If we send it again, any previous link will stop working
 
 emailChangeSuccess.title = We have received the new email address
 
-landlineChangeSuccess.title.change = Request to change landline number received
-landlineChangeSuccess.title.remove = Request to remove landline number received
-mobileChangeSuccess.title.change = Request to change mobile number received
-mobileChangeSuccess.title.remove = Request to remove mobile number received
+landlineChangeSuccess.title.change = You have asked to change the landline number
+landlineChangeSuccess.title.remove = You have asked to change the landline number
```

Comment:
If all the `remove`s are now the same as the `change`s could you delete them and update `ChangeSuccessController` to reflect this